apps/performance-tests/src/hierarchies/Filtering.test.ts

```diff
@@ -0,0 +1,107 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import { expect } from "chai";
+import { IModelDb, PhysicalElement, SnapshotDb } from "@itwin/core-backend";
+import { createNodesQueryClauseFactory, HierarchyFilteringPath } from "@itwin/presentation-hierarchies";
+import { Datasets } from "../util/Datasets";
+import { run } from "../util/TestUtilities";
+import { ProviderOptions, StatelessHierarchyProvider } from "./StatelessHierarchyProvider";
+import { createBisInstanceLabelSelectClauseFactory, ECClassHierarchyInspector, ECSchemaProvider } from "@itwin/presentation-shared";
+
+describe("filtering", () => {
+  const totalNumberOfFilteringPaths = 50000;
+  const physicalElementsSmallestIndex = 20;
```

Comment:
I believe this is not an index, but an id in decimal form.

Reply:
Renamed